src/routes/grantform/GrantFormSuccess.js

```diff
@@ -0,0 +1,47 @@
+import React, { PropTypes } from 'react';
+import withStyles from 'isomorphic-style-loader/lib/withStyles';
+import { connect } from 'react-redux';
+import s from './GrantFormSuccess.css';
+import Link from '../../components/Link';
+import history from '../../core/history';
+
+const GrantFormSuccess = ({ submission }) => {
+  if (!submission) {
+    if (history) {
+      history.push('/');
+    }
+    return null;
+  }
```

Comment:
This is a hack. Should probably be done in the route but I couldn't figure out how.